src/features/berth/berthFormPage/boatDetails/createBerthBoat/tabs/fragments/GdprNotes.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import ExternalLink from '../../../../../../../common/externalLink/ExternalLink';
```

Comment:
uhhh.. we should configure aliasis at some point this ../../..... does not look good. but not as part of this PR